portfolio/src/main/webapp/index.html

```diff
@@ -21,33 +21,33 @@ <h1>Naomi Amuzie's Portfolio</h1>
           society, and humanity. I love playing/watching volleyball and 
           reading op-eds. Fav TV Shows include Black Mirror, Sherlock, Big
           Little Lies, and The Handmaid's Tale. </p>
-      <img class="resize" src=/images/ADEC36D4-5273-4327-AB8D-2BBD98FA41E2.JPG/>
+      <img class="resize" alt = "selfie/photo of Naomi" src="/images/ADEC36D4-5273-4327-AB8D-2BBD98FA41E2.JPG">
```

Comment:
Nit: change `alt = "text"` to `alt="text"` here and other occurrences in the file to keep things consistent; the other tags like class and src don't have spaces around the equal sign.